addon/mixin.js

```diff
@@ -1,16 +1,26 @@
 /* eslint-disable ember/no-get */
 /* eslint-disable ember/no-new-mixins */
-import Ember from 'ember';
 import Mixin from '@ember/object/mixin';
 import { isArray } from '@ember/array';
 import { readOnly } from '@ember/object/computed';
-import { defineProperty, getProperties, set, get } from '@ember/object';
+import { defineProperty, getProperties, set, get, notifyPropertyChange } from '@ember/object';
+
+import { gte, lte } from 'ember-compatibility-helpers';
 
 import { aliasMethod, empty } from './helpers';
 
-const { notifyPropertyChange, meta } = Ember; // eslint-disable-line ember/new-module-imports
+import { meta as metaFor } from '@ember/-internals/meta';
```

Comment:
@lukemelia I don't know how much better it is to use the `-internals` import instead of the old `import Ember from 'ember';`

Reply:
Yeah, both seem not great. But then we're defining a mixin, which itself is not particularly modern so... 🤷

Reply:
so true! 🤣